server_manager/www/shadowbox_server.ts

```diff
@@ -260,8 +277,39 @@ export class ShadowboxServer implements server.Server {
     return await this.api.request<ServerConfigJson>('server');
   }
 
+  private async getSupportedExperimentalUniversalMetricsEndpoint(): Promise<boolean> {
+    if (this._supportedExperimentalUniversalMetricsEndpointCache !== null) {
+      return this._supportedExperimentalUniversalMetricsEndpointCache;
+    }
+
+    let result = false;
+
+    if (!this.api) return result;
+
```

Comment:
I think our style guide requires braces. Also, swap the order for readability.
```suggestion
    if (!this.api) {
      return false;
    }
    let result = false;
```